src/gc/none.cr

```diff
@@ -46,22 +48,24 @@ module GC
     Stats.new(zero, zero, zero, zero, zero)
   end
 
-  # :nodoc:
-  def self.pthread_create(thread : LibC::PthreadT*, attr : LibC::PthreadAttrT*, start : Void* -> Void*, arg : Void*)
-    LibC.pthread_create(thread, attr, start, arg)
-  end
-
-  # :nodoc:
-  def self.pthread_join(thread : LibC::PthreadT) : Void*
-    ret = LibC.pthread_join(thread, out value)
-    raise Errno.new("pthread_join") unless ret == 0
-    value
-  end
-
-  # :nodoc:
-  def self.pthread_detach(thread : LibC::PthreadT)
-    LibC.pthread_detach(thread)
-  end
+  {% unless flag?(:win32) %}
+    # :nodoc:
+    def self.pthread_create(thread : LibC::PthreadT*, attr : LibC::PthreadAttrT*, start : Void* -> Void*, arg : Void*)
+      LibC.pthread_create(thread, attr, start, arg)
+    end
+
+    # :nodoc:
+    def self.pthread_join(thread : LibC::PthreadT) : Void*
+                                                     ret = LibC.pthread_join(thread, out value)
```

Comment:
Indentation is off here.

Reply:
So it is, fixed.

Shame it's impossible to get the formatter to work for macrocode.